docs/book/src/tasks/automated-machine-management/healthchecking.md

```diff
@@ -209,15 +209,14 @@ This is useful for dynamically scaling clusters where the number of machines kee
 
 ## Skipping Remediation
 
-There are scenarios where remediation for a machine may be undesirable (eg. during cluster migration using `clusterctl move`). For such cases, MachineHealthCheck provides 2 mechanisms to skip machines for remediation.
+There are scenarios where remediation for a machine may be undesirable (eg. during cluster migration using `clusterctl move`). For such cases, MachineHealthCheck provides the following mechanisms to skip remediation.
 
-Implicit skipping when the resource is paused (using `cluster.x-k8s.io/paused` annotation):
-- When a cluster is paused, none of the machines in that cluster are considered for remediation.
-- When a machine is paused, only that machine is not considered for remediation.
-- A cluster or a machine is usually paused automatically by Cluster API when it detects a migration.
+- Users can skip remediation for a specific machine by setting the `cluster.x-k8s.io/skip-remediation` annotation on it.
+- Paused Machines (Machines with the `cluster.x-k8s.io/paused` annotation) are not considered for remediation.
+- If a specific MHC resource is paused (using `cluster.x-k8s.io/paused` annotation), it will stop to remediate the corresponding target machines.
```

Comment:
```suggestion
- If a specific MachineHealthCheck resource is paused (using `cluster.x-k8s.io/paused` annotation), it will stop to remediate the corresponding target machines.
```

Reply:
Updated.